styles/globals.css

```diff
@@ -20,3 +20,28 @@ a {
 * {
   box-sizing: border-box;
 }
+
+/* styles for top under construction banner start here*/
+
+.banner {
+  background: white;
```

Comment:
This style can be written easily using Tailwind(using `bg-white`). Please refrain from using `globals.css` for such.

Reply:
pushed in new commit